### PR TITLE
Create an overview section about Compass

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Add an overview section about Compass

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -6,3 +6,6 @@
 
 # These are the default owners for the whole content of the `compass` repository. The default owners are automatically added as reviewers when you open a pull request, unless different owners are specified in the file.
 * @PK85 @aszecowka @crabtree @pkosiec @kfurgol @tgorgol
+
+## README.md document
```
Comment (pkosiec) [inline CODEOWNERS]:
```suggestion
# README.md document
```

File: README.md [hunk 2]
```diff
@@ -5,9 +5,21 @@
...
+
+Compass allows for registering different types of applications and runtimes.
+These are the types of possible integration levels between an application and Compass:
+- basic - administrator manually provides API/Events Metadata to Compass. This type of integration is used mainly for simple use-case scenarios and doesn't support all features.
+- application - integration with Compass is built-in inside the application.
+- proxy - a highly application-specific proxy component provides the integration.
+- service -  a central service provides the integration for a class of applications. It manages multiple instances of these applications. You can integrate multiple services to support different types of applications.
+
+You can register any runtime, providing that it fulfills a contract with Compass and implements its flow. First, your runtime must get a trusted connection to Compass. It must also allow for fetching application definitions and using these applications in a given tenant. The example runtimes are Kyma (Kubernetes), CloudFoundry, Serverless, etc.
```
Comment (pkosiec) [inline README.md]: - Shouldn't we capitalize words like Runtime etc.?
- Maybe we should also link to Terminology?

File: installation/README.md [hunk 3]
```diff
@@ -2,34 +2,4 @@
...
-
-```bash
-./installation/scripts/testing.sh
-```
-
-Compass, as a part of Kyma, uses [Octopus](https://github.com/kyma-incubator/octopus/) for testing. Learn more in [Testing Kyma](https://kyma-project.io/docs/root/kyma#details-testing-kyma) section.
-
-> **NOTE:** After adding a new test, remember to add it also to `ClusterTestSuite` inside `testing.sh` script.
\ No newline at end of file
+Read [this](https://github.com/kyma-incubator/compass#installation) document to learn more about Compass installation and testing.
```
Comment (pkosiec) [inline installation/README.md]: Do we need this document? I would remove itd

File: README.md (hunk 2 above)
Comment (klaudiagrz) [reply README.md]: - We do not capitalize runtime in the MP docs. 
- There is a link to the documentation at the end of this section. I guess it's sufficient.

**Pull request merged**